### PR TITLE
 Leaderboard project

### PR DESCRIPTION
**Pull request by AhmadiGu**
### Leaderboard project

- use the gitflow
- configure the webpack
- configure the Linters
- add HTML code 
- add style for HTML
-

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @AhmadiGu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199

You have submitted a project that's not using the [updated readme template](https://github.com/microverseinc/readme-template). Kindly modify your readme file using the newly recommended template. Check the announcement channel on slack for more information about the new template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,15 +1,46 @@
...
+- Css
 
-```
-yarn build
-```
 
-to bundle your application
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
```
Comment (leonardodiasb) [inline README.md]: - [ ] Kindly give instructions to the user on how to use the application, like if the user should install the node dependencies with `npm i` and use the `npm run build` command to build the application, and after that he/she can open the live server. This section is missing the steps you mention in it.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @AhmadiGu, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of Gitflow. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by andgarzonmal — APPROVED** [review]
Hi @AhmadiGu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**